packages/web-react/src/components/Header/demo/HeaderExperimentalRTL.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Checkbox } from '../../Checkbox';
+
+const HeaderExperimentalRTL = () => {
+  const toggleRtl = () => {
+    const htmlElement = document.querySelector('html');
+
+    if (htmlElement?.hasAttribute('dir')) {
+      document.querySelector('html')?.removeAttribute('dir');
```

Comment:
You can use the `htmlElement` variable here, can't you? And the same on the line 14.

Reply:
you are right, fixed